CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # CHANGELOG
 
+## v2.0.1
+
+-- fix #176
+
+## v2.0.0
+
+-- Add new makara/internationalization support
```

Comment:
- And completely refactor
- And move config values from package.json to .yo-rc.json
- and fix some bugs if I remember right
- modernize and use new yeoman infrastructure